pkg/utils/string_test.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import (
+	"gotest.tools/assert"
+	"testing"
+)
+
+func Test_ToCamelCase(t *testing.T) {
+	assert.Equal(t, "Cont1aName", ToCamelCase("cont1a-name"))
+	assert.Equal(t, "Cont1aname", ToCamelCase("cont1aname"))
```

Comment:
I think it would be good to have TC for 2 digits together and apart, and also for Capital letter directly either side of number(s)